src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/updatelistingrequirements/UpdateListingRequirementsAboutToStartHandler.java

```diff
@@ -33,26 +34,36 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
 
         return callbackType.equals(CallbackType.ABOUT_TO_START)
             && (callback.getEvent() == EventType.UPDATE_LISTING_REQUIREMENTS);
-
     }
 
     @Override
-    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback, String userAuthorisation) {
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
+                                                          Callback<SscsCaseData> callback,
+                                                          String userAuthorisation) {
         if (!canHandle(callbackType, callback)) {
             throw new IllegalStateException("Cannot handle callback");
         }
 
         final SscsCaseData sscsCaseData = callback.getCaseDetails().getCaseData();
 
         if (isScheduleListingEnabled) {
+            log.info("Handling override fields update listing requirements event for caseId {}",
+                sscsCaseData.getCcdCaseId());
 
-            log.info("Handling override fields update listing requirements event for caseId {}", sscsCaseData.getCcdCaseId());
+            SchedulingAndListingFields schedulingAndListingFields = sscsCaseData.getSchedulingAndListingFields();
 
-            OverrideFields overrideFields = sscsCaseData.getSchedulingAndListingFields().getOverrideFields();
+            OverrideFields overrideFields = schedulingAndListingFields.getOverrideFields();
 
             if (isNull(overrideFields)) {
                 overrideFields = new OverrideFields();
-                sscsCaseData.getSchedulingAndListingFields().setOverrideFields(overrideFields);
+                schedulingAndListingFields.setOverrideFields(overrideFields);
+            }
+
+            OverrideFields listingValues = schedulingAndListingFields.getDefaultListingValues();
+
+            if (isNull(listingValues)) {
+                listingValues = new OverrideFields();
+                schedulingAndListingFields.setDefaultListingValues(listingValues);
             }

```

Comment:
Why is this here? DefaultListingValues can be left as null until it is set in hearings api

Reply:
Removed